src/app/containers/Timestamp/index.jsx

```diff
@@ -29,22 +30,60 @@ const formatTimestamp = dateObj => {
   return `${dayNumeric} ${monthLong} ${fullYear}`;
 };
 
-const TimestampContainer = ({ timestamp }) => {
-  const dateObj = new Date(timestamp);
+const isTenHoursAgo = milliseconds => {
+  const now = Date.now();
+  return now - milliseconds >= 10 * 60 * 60 * 1000;
+};
+
+const timestampWithPrefixUpdated = (datetime, milliseconds) => (
+  <Timestamp datetime={datetime} prefix="Updated">
+    {milliseconds}
+  </Timestamp>
+);
+
+const hasBeenUpdated = (updated, published) => updated !== published;
+
+const updatedTimestamp = (updated, published) => {
+  if (!hasBeenUpdated(updated, published)) {
+    return null;
+  }
 
-  if (!isValidDateTime(dateObj)) {
+  if (isTenHoursAgo(updated)) {
+    return timestampWithPrefixUpdated(
+      formatDateTime(new Date(updated)),
+      formatTimestamp(new Date(updated)),
+    );
+  }
+
+  return timestampWithPrefixUpdated(
+    formatDateTime(new Date(updated)),
+    relativeTime(updated),
+  );
+};
+
+const TimestampContainer = ({ updated, published }) => {
+  if (
+    !isValidDateTime(new Date(updated)) ||
+    !isValidDateTime(new Date(published))
+  ) {
     return null;
   }
 
+  const publishDate = new Date(published);
+
   return (
-    <Timestamp datetime={formatDateTime(dateObj)}>
-      {formatTimestamp(dateObj)}
-    </Timestamp>
+    <Fragment>
+      <Timestamp datetime={formatDateTime(publishDate)}>
+        {formatTimestamp(publishDate)}
+      </Timestamp>
+      {updatedTimestamp(updated, published)}
+    </Fragment>
```

Comment:
This file is quite tricky to read, it feels like it could be simpler? I'd aim for something like:

```jsx
return <Fragment>
    <Timestamp datetime={datetime(publishDate)}>
        {humanReadable(publishDate)}
    </Timestamp>
    { (publishDate === updatedDate)
        ? null;
        : (
            <Timestamp datetime={datetime(updatedDate)} prefix="Updated">
                
            </Timestamp>
        )
    }
</Fragment>;
```